provider/resource_keycloak_user_test.go

```diff
@@ -19,7 +19,7 @@ func TestAccKeycloakUser_basic(t *testing.T) {
 	realmName := "terraform-" + acctest.RandString(10)
 	username := "terraform-user-" + acctest.RandString(10)
 	attributeName := "terraform-attribute-" + acctest.RandString(10)
-	attributeValue := acctest.RandString(300)
+	attributeValue := acctest.RandString(250)
```

Comment:
these tests were failing for me locally, probably due to #132.  I am not sure why they pass in CI

Reply:
Yeah I nearly raised an Issue about the local test failures, but assumed I'd missed some setup. I can report back on whether the functionality is sufficient, but don't think I have sufficient context to investigate the existing test failures.

Reply:
Don't worry too much about the test failures, that's something I can look in to.

The changes I made will at least allow these tests to pass locally for me.  I'd be curious to see if they pass locally for you as well if you pull my branch down.

Reply:
FYI - these failed locally and passed in CI because my local setup uses postgres, which uses a `varchar(255)` column for user attributes, hence the failure.  CI uses in-memory h2 which does not appear to have that limitation.